packages/pieces/apps/src/lib/clickup/actions/spaces/get-spaces.ts

```diff
@@ -0,0 +1,23 @@
+import { createAction, Property, HttpMethod, getAccessTokenOrThrow } from "@activepieces/framework";
+import { clickupCommon, callClickUpApi } from "../../common";
+
+
+export const getClickupSpaces = createAction({
+	name: 'get_spaces',
+	description: 'Gets spaces in a ClickUp workspace',
+	displayName: 'Get Spaces',
+	props: {
+		authentication: clickupCommon.authentication,
+		team_id: Property.ShortText({
```

Comment:
The team_id refers to workspace_id as well in clickup, we can use the dropdown `workspace_id` in common file here instead.